pages/solid-events.md

```diff
@@ -18,8 +18,6 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
 
 - Interested in speaking at an upcoming Solid World? You can fill out the [Solid World Speaker form](https://es1cz4pb7oi.typeform.com/to/nietD34f).
 
-- Have a topic suggestion for an upcoming Solid World? You can fill out the [Solid World Topic Suggestions form](https://es1cz4pb7oi.typeform.com/to/yQNMvK0N).
```

Comment:
The new link is: https://es1cz4pb7oi.typeform.com/to/nietD34f?utm_source=SolidWorld
Sorry I cannot make a direct suggestion on deleted stuff... it seems like.

Reply:
We have that link in the point above it, for being a speaker